src/main/resources/docs/quick-start.md

```diff
@@ -0,0 +1,532 @@
+9. Now, find **start.kt** in your project - you can either tap the shift key twice to open quick search or follow the path **todolist > src > kotlin > start.kt**
+10. Right click on **start.kt** and select option to **Run**
+11. Once run is successful, click [http://localhost:8080/](http://localhost:8080/) to view the project in your browser
+
+</div>
+
+**TADA!** 🎉 - you are viewing the basic Alpas start screen. But, there is still a bit more to do to create your to-do list. Now that we have Alpas successfully installed and running
+on your machine, let's create your to-do list! 
+
+<a name="setup-database"></a>
+### [Step 3: Setup your database](#setup-database)
+Let's first start by creating a new database on your local machine. If you haven't already installed MySQL Server and have it running on your machine, refer to the [What you'll need](#need) section for links to set up MySQL. 
+
+<a name="sequel-pro"></a>
+#### [Create the to-do database in Sequel Pro](#sequel-pro)
+
+<div class="ordered-list">
+
+1. Open **Sequel Pro**
+2. Add a new connection 
+3. Type **127.0.0.1** into Host 
+4. Type in **root** for the Username
+5. If you entered a password when installing MySQL, add in the password; otherwise, leave it blank
+6. Click **Connect**
+7. Once connected to your local MySQL server, go to **Database > Add Database**
+8. Create a new database named **todolist**
+
+</div>
+
+That's all for now in Sequel Pro. We will add tables and fields during the next steps. 
+
+<a name="connect-db"></a>
+#### [Connect your to-do project to MySQL database](#connect-db)
+
+<div class="ordered-list">
+
+1. Go back to the to-do project up in IntelliJ
+2. Navigate to the **.env** file under the project root folder
+3. In .env, update `DB_PASSWORD=secret` to replace 'secret' with your password; leave blank if you did not setup your MySQL database with a password
+4. Update `DB_DATABASE` to point to `todolist`
+5. Go to the **DatabaseConfig** file in **todolist > src > main > kotlin > configs** 
+6. Uncomment `// addConnections(env)` by removing `//`
+7. **Run** the project 
+
+</div>
+
+If you run and are able to successfully access [http://localhost:8080/](http://localhost:8080/), then congrats! You have successfully connected the project to your MySQL database. 
+
+<a name="setup-entity"></a>
+### [Step 4: Setup your task entity](#setup-entity)
+
+<div class="ordered-list">
+
+1. In IntelliJ's terminal, type `./alpas make:entity task`<span class="clipboard" data-clipboard-text='./alpas make:entity task'></span> and tap return - this will create a new entity 'Task' and corresponding table
+ under **src > main > kotlin > entities** folder
```

Comment:
Too many `In IntelliJ's terminal`. I think we can just say `From the terminal` or something like that